phsa-adapter/src/Controllers/ReceiveController.cs

```diff
@@ -21,7 +18,6 @@ public ReceiveController(ILogger<ReceiveController> logger)
         }
 
         [HttpPost]
-        [BasicAuthorization]
```

Comment:
The basic auth code is needed for the non-FHIR inbound payload from PHSA

Reply:
reverted, changed to Authorize at controller and enforce basic authentication policy